src/client/components/login/login.stories.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Login from './login.component';
+import { action } from '@storybook/addon-actions';
+import { withKnobs, text, select } from '@storybook/addon-knobs';
+
+export default {
+  title: 'ContainerLayouts/Login',
```

Comment:
What do you plan to put in the category "ContainerLayouts"? I think it's a bit of a strange name. You can just scratch the category and give it a title for now. We can group things later.

```suggestion
  title: 'Login',
```